sim/midas/src/main/cc/simif_emul.cc

```diff
@@ -38,76 +38,17 @@ simif_emul_t::simif_emul_t(const TargetConfig &config,
     (*slave.rbegin())->init(memsize, 64);
   }
 
-  using namespace std::placeholders;
-  auto mmio_read_func = std::bind(&simif_emul_t::read, this, _1);
-  auto mmio_write_func = std::bind(&simif_emul_t::write, this, _1, _2);
-
+  // Set up the managed stream.
   if (auto conf = config.cpu_managed) {
```

Comment:
I know that this is a pattern for using `std::optional`, but I don't like the way it looks. can you at least add a comment here saying that these are `std::optional` ?

Reply:
Done, made the type explicit.